cathedral-town/mod.json

```diff
@@ -15,6 +15,16 @@
 		"1.0.1"   : [ "Special buildings support" ],
 		"1.0.0"   : [ "Initial release" ]
     },
+    
+    "german" : {
+		"name" : "Kathedrale",
+		"description" : "Die Kathedrale ist eine Mod, die von Ben Yan neu erstellt wurde",
```

Comment:
maybe keep original name in ( ) so people dont get confused about international mod names?

Reply:
I think we should handle this in code rather in translation.
So if you think that we should show original name, I can make a change and always show original name in mod description.
Translations should not bother with things like these.